packages/int-mask/src/defaults.ts

```diff
@@ -0,0 +1,105 @@
+import { IntMaskOptions } from './intMask';
+
+export const defaultOptions: IntMaskOptions = {
```

Comment:
Initial way i had our dummy maps setup were wrong for testing purposes. There are required a certain set of characters and indexes for each number, position.